### DISCUSSION TITLE
Encourage the engine developers to create games in Godot

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4694] [General Discussions]
I really believe it would be better for the engine if the engine devs will create and release games.
I started learning Godot making some simple plugins in GDScript.
Recently I finally started making games.
And you know what, I found that I used only a few of my plugins.
And the lesson I have taught is if you didn't make a complete game, you don't get "the whole picture" and don't really know what is lacking.
Well this can ofc depend on the project, but still there will be some things you will certainly improve because you don't want to have these problems in any project anymore.

For example now I think I would love to have a tool (maybe a plugin) that have extended features for searching scenes and resources.
For example "find all scenes in the project that contain a node with type Animation player".
Or "find all Resources that have a property with a certain value".
Sometimes I waste much time trying to find a scene even if they are named properly.

How to encourage the engine developers to create games? Idk
But I think this is important

**Comment by Calinou** [comment]
While I agree that a game engine works best when its developers dogfood it, I think this is unfortunately not practical for the Godot developers to do. We already have way more work to do than we can actually handle right now :slightly_smiling_face: 

Even developing a vertical slice (like a more complete TPS demo) would take a lot of time away from developing features and fixing bugs in the engine.

**Reply by Barugon** [reply]
How about an official repository under the godotengine organization for a community developed 3D demo game designed to show off the Godot engine?

**Reply by theraot** [reply]
@Barugon Is this what you are looking for: https://github.com/godotengine/godot-demo-projects/tree/master/3d ?

Note: I notice the readmes still mention GLES2 and GLES3, however, I see there have been updates to Godot 4.

Errata: I didn't pay attention to the fact that you said "community developed". these are official.

Addendum: See https://godotengine.org/showcase/

**Comment by theraot** [comment]
I believe the community bridges this gap. We can make games, make plugins, report issues, make proposal, and even contribute code. Each on their own capacity, of course. So, if you want people who make games and contribute to Godot, look at the community. *In fact, there are tons of proposals and pull requests already, which does not help the core developers to find time to make games.*
